xml/admin_saltcluster.xml

```diff
@@ -1141,14 +1141,29 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </step>
    <step>
     <para>
-     If the &adm; is a &smaster; at the same time, stop
-     <systemitem class="daemon">salt-master</systemitem>
-     <systemitem class="daemon">salt-minion</systemitem> services on the old
-     &adm;:
+     If the old &smaster; is also the only administration node in the cluster
```

Comment:
Shouldn't be `... the only &adm; node`?

Reply:
Not according to @smithfarm - https://bugzilla.suse.com/show_bug.cgi?id=1176326#c22 See the suggested change to:

```2. If the old Salt Master node is also THE ONLY ADMIN NODE IN THE CLUSTER (see New Section Explaining Admin Nodes for more information), then manually move '/etc/ceph/ceph.client.admin.keyring' and '/etc/ceph/ceph.conf' to the new Salt Master. ```

Reply:
Ahh, you are right... we are moving the salt-master.